.github/ISSUE_TEMPLATE/netbeans_bug_report.yml

```diff
@@ -54,15 +54,28 @@ body:
         Please make sure you provide a reproducible step-by-step case of how to reproduce the problem
         as minimally and precisely as possible. Remember that non-reproducible issues may be closed or
         converted into discussions. If we cannot reproduce an issue we cannot fix it!
-  - type: checkboxes
+  - type: dropdown
     attributes:
       label: Did this work correctly in an earlier version?
       description: >
         It's important to know whether bugs have been introduced by recent changes. If this used
         to work, tick this and make sure you've specified the last version that worked correctly
         in your issue description.
+      multiple: false
       options:
-        - label: This used to work!
+        - "No"
```

Comment:
I deliberately left out versions when porting from Airflow because we'll have to keep updating them.  Still, pros and cons to that.  Isn't it missing 9.x?

Reply:
I figured that was the case, but it is useful information when attempting the track down where the regression started.
Yes, it is missing 9.x.

I suppose maybe it is better to have yes/no then have a text field to enter last version it worked.... or just text field for last version it worked?

Reply:
@oyarzun I agree. It does ask people to specify in the text, but better to try and capture - it was purely to reduce need to update the templates for each new release. @mbien also mentioned on dev@ about capturing the actual version in the first dropdown, so if we do one we should do the other.  Can we do one review and set of changes in a couple of weeks?  As we move to switch off JIRA.  Happy for that to happen in this PR.

Reply:
yes I think having a "latest" selection could cause problems. The user who files the issue might not have the same idea of "latest" as a dev has. E.g latest could be the latest in the package manager of a distro, not the latest available version.

Combo box is fine but would require periodic updates (as neil mentioned) - I would be also ok with a plain old text field. Not sure what others think about this.

Reply:
@mbien the reason for not having a text field in the first place was to try and pre-empt reports on older versions from being made. I'd like to keep that if we can, so updating at releases is probably the right way forward - not a huge addition given everything else that needs doing.

So at release, for the other (version) dropdown, we could have `NetBeans 13`, `NetBeans 14 release candidate`, `Daily build`?

Reply:
@neilcsmith-net sounds good. I like your thinking of motivating users to update by leaving out older versions :)